packages/x-components/src/x-modules/queries-preview/components/__tests__/query-preview.spec.ts

```diff
@@ -119,9 +120,24 @@ describe('query preview', () => {
     expect(wrapper.emitted('load')?.[0]).toEqual(['shoes']);
   });
 
+  it('does not remove the query before destroy the component', () => {
```

Comment:
before destroying or in the beforeDestroy component's hook

Reply:
sorry, and add the value of the cache prop in the test statement to make sure we are testing cache is working ok